firmware/boron-ins-fsm/test/consoleFunctionTests.cpp

```diff
@@ -143,6 +143,60 @@ SCENARIO("Change_IM21_Door_ID", "[change door id]") {
     }
 }
 
+SCENARIO("Set Window Time", "[window time]") {
+    GIVEN("A starting window time of 1 minute") {
+        state0_window_time = 60000;
+
+        WHEN("the function is called with 'e'") {
+            int returnFlag = window_size_set("e");
+
+            THEN("the initial timer value should remain the same") {
+                REQUIRE(state0_window_time == 60000);
+            }
+
+            THEN("the function should return the input") {
```

Comment:
I think the description of this "THEN" should be updated. When I read "the function should return the input", it sounds like it was supposed to return "e", which is not the case.

Reply:
I agree. I'll change all of the other instances of this in other SCENERIOS as well